lib/zones/main_street/main_street2.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:flutter/material.dart';
+import 'package:provider/src/provider.dart';
+import 'package:zero_city/state/main_street_state.dart';
+import 'package:zero_city/state/planning_lab_state.dart';
+import 'package:zero_city/utils/Graphics.dart';
+import 'package:zero_city/zones/main_street/main_street3.dart';
+import 'package:zero_city/zones/planning_lab/planning_lab2.dart';
+import 'package:zero_city/text_types/mission_body.dart';
+import 'package:zero_city/text_types/mission_title.dart';
+
+class MainStreet2 extends StatefulWidget {
+  const MainStreet2({Key? key}) : super(key: key);
+
+  @override
+  State<MainStreet2> createState() => MainStreetState2();
+}
+
+class MainStreetState2 extends State<MainStreet2> {
+  // Fel svar snackbar.
+  SnackBar snack = const SnackBar(
```

Comment:
Can this be displayed on text submit instead of (grey) button press? Might be easier for UI usability